src/routes/course.routes.js

```diff
@@ -1,6 +1,6 @@
 const router = require("express").Router();
 const { getAllCourses } = require ('../controllers/test.controller'); 
```

Comment:
should be `const { getAllCourses } = require ('../controllers/admin.controller');`  ?